docs/v3.x/guides/custom-admin.md

```diff
@@ -114,7 +154,7 @@ And tada, the date will now display with the new format.
 
 ## Build the new admin
 
-Well now you have the admin panel you want. But during all the process, the admin panel was updated on the run time because of the command `yarn develop --watch-admin`.
+Well now you have the admin panel you want. But during all the process, the admin panel was updated on the run time because of the command `yarn develop --watch-admin` / `npm run develop -- --watch-admin`.
```

Comment:
```suggestion
Well now you have the admin panel you want. But during all the process, the admin panel was updated on the run time because of the command `yarn develop --watch-admin` or `npm run develop -- --watch-admin`.
```